src/Life/Github.hs

```diff
@@ -30,8 +30,9 @@ import Life.Configuration (LifeConfiguration (..), lifeConfigMinus, parseRepoLif
 import Life.Message (chooseYesNo, errorMessage, infoMessage, warningMessage)
 import Life.Shell (lifePath, relativeToHome, repoName, ($|))
 
-newtype Owner = Owner { getOwner :: Text } deriving (Show)
-newtype Repo  = Repo  { getRepo  :: Text } deriving (Show)
+newtype Owner  = Owner  { getOwner  :: Text } deriving (Show)
+newtype Repo   = Repo   { getRepo   :: Text } deriving (Show)
+newtype Branch = Branch { getBranch :: Text } deriving (Show)
```

Comment:
Oh, it's a good time to refactor existing code according to modern style-guide suggestions. Let's replace `get` prefix in `newtypes` with `un` prefix.